massgov/mayflower
# DP-32013 [Mobile review] Links in contact section style fix

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

- Add an underline to links (not decorative links) in the contact component matching the RT link style
<img width="1001" alt="Screenshot 2024-04-26 at 10 25 06 AM" src="https://github.com/massgov/mayflower/assets/9633303/62d409df-1d5d-429f-ac5c-9ed73b8d21f4">

- Modify the underline to flow as a link splits to multiple lines (= each line of the link has an underline)
- No visual change other than the 1px underline at normal state of the links

## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-32013)
- [Github issue]()

## Steps to Test


1. Go to the sample org pages.
2. Find that the links (not 'decorative links', e.g., phone numbers, emails) in the contact component have 1px underline at normal state.
3. Hover over to the links and the underline width changes to 3px, but they don't push down the content below.
4. When the links split into multiple lines with a small screen width (320px is the smallest screen width in the currently available devices), the underline stays on each line of the links.

Like this (see the email address):
<img width="321" alt="Screenshot 2024-04-26 at 10 24 36 AM" src="https://github.com/massgov/mayflower/assets/9633303/ea8d2a6e-cd6a-4fe7-a188-ccf8c656c2f3">
 
Not like this:
<img width="147" alt="inline" src="https://github.com/massgov/mayflower/assets/9633303/33e77905-39b5-4fb3-808b-ec5d8d5eb298">

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
